mmv1/third_party/terraform/.teamcity/components/build_config_package.kt

```diff
@@ -49,26 +44,41 @@ class packageDetails(name: String, displayName: String, environment: String, bra
             }
 
             params {
-                TerraformAcceptanceTestParameters(parallelism, "TestAcc", "12", "us-central1", "")
+                ConfigureGoogleSpecificTestParameters(environmentVariables)
+                TerraformAcceptanceTestParameters(parallelism, "TestAcc", "12", "", "")
                 TerraformAcceptanceTestsFlag()
                 TerraformCoreBinaryTesting()
                 TerraformShouldPanicForSchemaErrors()
                 ReadOnlySettings()
-                WorkingDirectory(path, packageName)
+                WorkingDirectory(path)
             }
 
-            triggers {
-                RunNightly(nightlyTestsEnabled, startHour, daysOfWeek, daysOfMonth, branchRef)
+            // triggers are not set here because the dependent Post-Sweeper build causes these builds
+            // to be triggered; if these have their own trigger as well then they will run twice.
+            // This problem can be migitated via `reuseBuilds` in dependencies on Post-Sweeper
+            // but that mitigation doesn't appear to scale well as there are more packages
+
+            dependencies {
+                // Acceptance tests need the Pre-Sweeper step to have completed first
+                snapshot(RelativeId(preSweeperBuildConfigId)) {
```

Comment:
All builds for (service) packages are created with a dependency on the pre-sweeper. This ensures that the pre-sweeper must run before the acceptance tests, and acc tests don't run while the sweeper is active.

Reply:
Also, `failureConditions` below this code is where we ensure build configs fail if they run for a long time